utxo/utxo.go

```diff
@@ -569,14 +569,14 @@ func (uv *UtxoVM) GenerateRootTx(js []byte) (*pb.Transaction, error) {
 //输入: 转账人地址、公钥、金额、是否需要锁定utxo
 //输出：选出的utxo、utxo keys、实际构成的金额(可能大于需要的金额)、错误码
 func (uv *UtxoVM) SelectUtxos(fromAddr string, fromPubKey string, totalNeed *big.Int, needLock, excludeUnconfirmed bool) ([]*pb.TxInput, [][]byte, *big.Int, error) {
-	uv.clearExpiredLocks()
-	curLedgerHeight := uv.ledger.GetMeta().TrunkHeight
-	willLockKeys := make([][]byte, 0)
-	utxoTotal := big.NewInt(0)
-	foundEnough := false
-	// 先从cache里找找，不够再从leveldb找,因为leveldb prefix scan比较慢
-	cacheKeys := map[string]bool{}
-	txInputs := []*pb.TxInput{}
+	var (
```

Comment:
It's not our code style to declare and initialize local variables using "var". instead, we use short variable declarations.

Reply:
Means that your code style is not allowed to be modified, is it? @yucaowang

Reply:
I think it's better to keep consistent with the code repo if there isn't any strong reason.

Reply:
Understand, in addition, the above Mac problem is also caused by the code style? @yucaowang

Reply:
I'm not sure, but I have triggered the CI build again, let's see if it's a occasional problems.

Reply:
Thanks alot.

Reply:
CI build passed, please go ahead.

Reply:
@yucaowang looks like it's all passed.

Reply:
@yucaowang The code style has been restored. Is it up to your requirements now?